demo/bfcachetest/index.js

```diff
@@ -0,0 +1,34 @@
+import { anAtom, aMap } from "../index.js";
```

Comment:
Use prettier in text editor

Reply:
I couldn't find prettier.config. Not sure where the problem is... Besides semicolon at the end of the line.
I am using vscode, and I have prettier+ as plugin

Reply:
Ah, It was in package.json... Fixed code style